src/librustc_passes/layout_test.rs

```diff
@@ -84,7 +85,7 @@ impl LayoutTest<'tcx> {
                         sym::debug => {
                             self.tcx.sess.span_err(
                                 item.span,
-                                &format!("layout debugging: {:#?}", *ty_layout),
+                                &format!("layout debugging for type {:?}: {:#?}", ty, *ty_layout),
```

Comment:
Wondering if maybe this should be "layout for type" without "debugging".
Or `"layout_of({}) = {:#?}"` :P.

Reply:
(but this PR is rolled up so this is for another time)

Reply:
Would be good to have some backticks if not that.